Source/Core/VideoBackends/Software/SWmain.cpp

```diff
@@ -41,7 +41,7 @@
 
 #define VSYNC_ENABLED 0
 
-static volatile u32 s_swapRequested = false;
+static std::atomic<bool> s_swapRequested;
 
 static volatile struct
```

Comment:
Oh, then I'm sorry. This variables don't need the volatile flag. They should be protected by s_swapRequested. /But/ Video_BeginField doesn't check if s_swapRequested is already reseted. Maybe we also need a busy loop there (VideoCommon does so)